src/collision.cpp

```diff
@@ -99,17 +99,18 @@ Entity* entityClicked(bool* clickedOnGUI, bool clickCheckOverride, int player, E
 		}
 		if ( openedChest[player] )
 		{
-			if ( mx > getChestGUIStartX(player)
-				&& mx < getChestGUIStartX(player) + inventoryChest_bmp->w
-				&& my > getChestGUIStartY(player)
-				&& my < getChestGUIStartY(player) + inventoryChest_bmp->h )
-			{
-				if ( clickedOnGUI )
-				{
-					*clickedOnGUI = true;
-				}
-				return NULL;    //Click falls inside the chest inventory GUI.
-			}
+			// TODO UI: REPLACE
```

Comment:
Does this still need to be handled? Is it only the buttons below the window that need this, or the whole window?

Reply:
Assuming that the click detection on frames solves most of the detection, floating widgets like Buttons may/may not need some extras added. collision.cpp/clickdescription.cpp are the 2 spots that need this deep dive